
Xvnc Free Edition 4.1.1 - built Feb 25 2015 23:02:21
Copyright (C) 2002-2005 RealVNC Ltd.
See http://www.realvnc.com for information on VNC.
Underlying X server release 40300000, The XFree86 Project, Inc


Fri Nov 29 11:28:00 2019
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on port 5905
 vncext:      created VNC server for screen 0
error opening security policy file /etc/X11/xserver/SecurityPolicy
Could not init font path element /usr/X11R6/lib/X11/fonts/Type1/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/Speedo/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/misc/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/75dpi/, removing from list!
Could not init font path element /usr/X11R6/lib/X11/fonts/100dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/75dpi/, removing from list!
Could not init font path element /usr/share/fonts/X11/100dpi/, removing from list!
xrdb: No such file or directory
xrdb: can't open file '/home/is28/zhirnova28/.Xresources'
/usr/bin/startxfce4: X server already running on display :5
gpg-agent[65184]: WARNING: "--write-env-file" is an obsolete option - it has no effect
gpg-agent: a gpg-agent is already running - not starting a new one

(xfce4-session:65177): xfce4-session-WARNING **: 11:28:03.225: gpg-agent returned no PID in the variables

(xfce4-session:65177): xfce4-session-WARNING **: 11:28:03.226: xfsm_manager_load_session: Something wrong with /home/is28/zhirnova28/.cache/sessions/xfce4-session-virt-1:5, Does it exist? Permissions issue?

(xfwm4:65187): xfwm4-WARNING **: 11:28:03.252: The display does not support the XRender extension.
(uint32 2,)

(xfwm4:65187): xfwm4-WARNING **: 11:28:03.253: The display does not support the XComposite extension.

(xfwm4:65187): xfwm4-WARNING **: 11:28:03.253: The display does not support the XDamage extension.

(xfwm4:65187): xfwm4-WARNING **: 11:28:03.253: The display does not support the XFixes extension.

(xfwm4:65187): xfwm4-WARNING **: 11:28:03.253: Compositing manager disabled.

** (light-locker:65196): ERROR **: 11:28:03.256: Environment variable XDG_SESSION_PATH not set. Is LightDM running?

(polkit-gnome-authentication-agent-1:65198): GLib-CRITICAL **: 11:28:03.275: g_variant_new_string: assertion 'string != NULL' failed

(polkit-gnome-authentication-agent-1:65198): polkit-gnome-1-WARNING **: 11:28:03.277: Failed to register client: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files
xfwm4-Message: 11:28:03.299: Unsupported keyboard modifier '<Super>Tab'

(xfsettingsd:65200): xfsettingsd-CRITICAL **: 11:28:03.362: RANDR extension is too old, version 1.1. Display settings won't be applied.
Xlib:  extension "XInputExtension" missing on display ":5.0".

(xfsettingsd:65200): xfsettingsd-CRITICAL **: 11:28:03.362: XI is not present.

(xfsettingsd:65200): xfsettingsd-CRITICAL **: 11:28:03.363: Failed to initialize the Xkb extension.

(xfsettingsd:65200): xfsettingsd-CRITICAL **: 11:28:03.371: Failed to initialize the Accessibility extension.
xfwm4-Message: 11:28:03.459: Unsupported keyboard modifier '<Super>Tab'

(tracker-extract:65211): Tracker-CRITICAL **: 11:28:03.515: Could not request DBus name 'org.freedesktop.Tracker1.Miner.Extract': D-Bus service name:'org.freedesktop.Tracker1.Miner.Extract' is already taken, perhaps the application is already running?

(tracker-miner-apps:65260): Tracker-CRITICAL **: 11:28:03.542: Could not request DBus name 'org.freedesktop.Tracker1.Miner.Applications': D-Bus service name:'org.freedesktop.Tracker1.Miner.Applications' is already taken, perhaps the application is already running?

(nm-applet:65267): Gtk-WARNING **: 11:28:03.554: Can't set a parent on widget which has a parent

(tracker-miner-fs:65201): Tracker-CRITICAL **: 11:28:03.563: Could not request DBus name 'org.freedesktop.Tracker1.Miner.Files': D-Bus service name:'org.freedesktop.Tracker1.Miner.Files' is already taken, perhaps the application is already running?

(nm-applet:65267): Gdk-CRITICAL **: 11:28:03.691: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed

Fri Nov 29 11:28:07 2019
 Connections: accepted: 0.0.0.0::50156
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)
/usr/lib/python3/dist-packages/blueman/plugins/applet/AppIndicator.py:8: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3 as girAppIndicator
ERROR:dbus.proxies:Introspect error on org.bluez:/org/bluez: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.6669" (uid=1271 pid=65265 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.6669" (uid=1271 pid=65265 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)
ERROR:dbus.proxies:Introspect error on org.bluez:/: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Rejected send message, 2 matched rules; type="method_call", sender=":1.6669" (uid=1271 pid=65265 comm="/usr/bin/python3 /usr/bin/blueman-applet " label="unconfined") interface="org.freedesktop.DBus.Introspectable" member="Introspect" error name="(unset)" requested_reply="0" destination="org.bluez" (bus)

(wrapper-1.0:65304): Gdk-WARNING **: 11:28:10.486: /build/gtk+2.0-AoeliP/gtk+2.0-2.24.32/gdk/x11/gdkdrawable-x11.c:952 drawable is not a pixmap or window

(wrapper-1.0:65304): Gdk-WARNING **: 11:28:10.487: /build/gtk+2.0-AoeliP/gtk+2.0-2.24.32/gdk/x11/gdkdrawable-x11.c:952 drawable is not a pixmap or window

Fri Nov 29 11:28:12 2019
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565

(polkit-gnome-authentication-agent-1:65198): polkit-gnome-1-WARNING **: 11:29:38.289: Couldn't open user icon: Failed to open file '/home/adom/.face': No such file or directory

(polkit-gnome-authentication-agent-1:65198): polkit-gnome-1-WARNING **: 11:29:38.290: Couldn't open user icon: Failed to open file '/home/test/.face': No such file or directory
Gtk-Message: 11:29:38.316: GtkDialog mapped without a transient parent. This is discouraged.

(polkit-gnome-authentication-agent-1:65198): polkit-gnome-1-WARNING **: 11:33:43.336: Couldn't open user icon: Failed to open file '/home/adom/.face': No such file or directory

(polkit-gnome-authentication-agent-1:65198): polkit-gnome-1-WARNING **: 11:33:43.337: Couldn't open user icon: Failed to open file '/home/test/.face': No such file or directory
Gtk-Message: 11:33:43.342: GtkDialog mapped without a transient parent. This is discouraged.

(Thunar:65193): thunar-WARNING **: 12:15:08.629: Content type loading failed for .goutputstream-KSGCC0: Error when getting information for file “/home/is28/zhirnova28/zad5/.goutputstream-KSGCC0”: No such file or directory

(Thunar:65193): thunar-WARNING **: 12:15:17.749: Content type loading failed for .goutputstream-YAEFC0: Error when getting information for file “/home/is28/zhirnova28/zad5/.goutputstream-YAEFC0”: No such file or directory

(gedit:20152): Gtk-WARNING **: 12:16:03.973: Calling org.xfce.Session.Manager.Inhibit failed: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Method "Inhibit" with signature "susu" on interface "org.xfce.Session.Manager" doesn't exist


(Thunar:65193): Gdk-WARNING **: 12:18:26.859: gdk_window_set_icon_list: icons too large
Xlib:  extension "XINERAMA" missing on display ":5.0".
func=xmlSecCheckVersionExt:file=xmlsec.c:line=188:obj=unknown:subj=unknown:error=19:invalid version:mode=abi compatible;expected minor version=2;real minor version=2;expected subminor version=25;real subminor version=26

(soffice:20588): GLib-GObject-WARNING **: 12:18:49.613: ../../../../gobject/gsignal.c:3492: signal name 'selection_changed' is invalid for instance '0x55b7dc290120' of type 'OOoAtkObjCompTxt'
func=xmlSecCheckVersionExt:file=xmlsec.c:line=188:obj=unknown:subj=unknown:error=19:invalid version:mode=abi compatible;expected minor version=2;real minor version=2;expected subminor version=25;real subminor version=26

(Thunar:65193): Gdk-WARNING **: 12:20:18.025: gdk_window_set_icon_list: icons too large

(Thunar:65193): Gdk-WARNING **: 12:20:25.041: gdk_window_set_icon_list: icons too large

(Thunar:65193): Gdk-WARNING **: 12:28:15.858: gdk_window_set_icon_list: icons too large

Fri Nov 29 12:29:52 2019
 XserverDesktop: Added unknown keysym 0x6d0 to keycode 255
 XserverDesktop: Added unknown keysym 0x6cf to keycode 254
 XserverDesktop: Added unknown keysym 0x6cc to keycode 253
 XserverDesktop: Added unknown keysym 0x6c9 to keycode 252
 XserverDesktop: Added unknown keysym 0x6ce to keycode 251

Fri Nov 29 12:29:53 2019
 XserverDesktop: Added unknown keysym 0x6c1 to keycode 250
xfwm4-Message: 12:29:53.400: Unsupported keyboard modifier '<Super>Tab'

Fri Nov 29 12:29:57 2019
 XserverDesktop: Added unknown keysym 0x6d3 to keycode 249
xfwm4-Message: 12:29:57.824: Unsupported keyboard modifier '<Super>Tab'

Fri Nov 29 12:29:59 2019
 XserverDesktop: Added unknown keysym 0x6d4 to keycode 248
xfwm4-Message: 12:30:00.112: Unsupported keyboard modifier '<Super>Tab'

Fri Nov 29 12:30:00 2019
 XserverDesktop: Added unknown keysym 0x6d8 to keycode 247
 XserverDesktop: Added unknown keysym 0x6d1 to keycode 246
xfwm4-Message: 12:30:00.680: Unsupported keyboard modifier '<Super>Tab'

Fri Nov 29 12:30:52 2019
 XserverDesktop: Added unknown keysym 0x6d7 to keycode 245
 XserverDesktop: Added unknown keysym 0xf3 to keycode 244
xfwm4-Message: 12:30:52.575: Unsupported keyboard modifier '<Super>Tab'

Fri Nov 29 12:36:00 2019
 Connections: closed: 0.0.0.0::50156 (Clean disconnection)
 SMsgWriter:  framebuffer updates 11231
 SMsgWriter:    copyRect rects 4345, bytes 69520
 SMsgWriter:    hextile rects 43694, bytes 109635067
 SMsgWriter:    raw bytes equivalent 927850286, compression ratio 8.463079

Fri Nov 29 12:36:38 2019
 Connections: accepted: 0.0.0.0::50261
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)

Fri Nov 29 12:36:44 2019
 VNCSConnST:  Server default pixel format depth 16 (16bpp) little-endian rgb565
 VNCSConnST:  Client pixel format depth 16 (16bpp) little-endian rgb565

(Thunar:65193): Gdk-WARNING **: 12:36:52.033: gdk_window_set_icon_list: icons too large

Fri Nov 29 12:37:22 2019
 Connections: closed: 0.0.0.0::50261 (Clean disconnection)
 SMsgWriter:  framebuffer updates 172
 SMsgWriter:    copyRect rects 84, bytes 1344
 SMsgWriter:    hextile rects 930, bytes 1352327
 SMsgWriter:    raw bytes equivalent 15108464, compression ratio 11.172197
xfce4-session: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.
xfwm4: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.
xfce4-panel: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.
xfdesktop: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.
Thunar: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.
Gdk-Message: 11:32:29.475: polkit-gnome-authentication-agent-1: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

Gdk-Message: 11:32:29.475: update-notifier: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

Gdk-Message: 11:32:29.481: xfce4-terminal: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

xfsettingsd: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

(nm-applet:65267): Gdk-WARNING **: 11:32:29.597: GdkWindow 0x220000a unexpectedly destroyed
Gdk-Message: 11:32:29.597: nm-applet: Fatal IO error 25 (Inappropriate ioctl for device) on X server :5.0.

wrapper-1.0: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.
wrapper-1.0: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.
Gdk-Message: 11:32:29.598: gedit: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

Gdk-Message: 11:32:29.598: evolution-alarm-notify: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

X IO Error
Gdk-Message: 11:32:29.598: blueman-applet: Fatal IO error 11 (Resource temporarily unavailable) on X server :5.0.

/usr/share/system-config-printer/applet.py:45: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
system-config-printer-applet: failed to start NewPrinterNotification service
system-config-printer-applet: failed to start PrinterDriversInstaller service: org.freedesktop.DBus.Error.AccessDenied: Connection ":1.6667" is not allowed to own the service "com.redhat.PrinterDriversInstaller" due to security policies in the configuration file
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 11 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 15 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 16 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 20 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 24 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 26 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 28 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 30 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 34 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 36 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 40 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 41 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 45 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 47 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 49 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 53 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 55 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 59 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 60 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 64 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 66 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 67 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 71 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 73 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 74 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 78 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 80 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 82 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 86 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 87 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 91 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 93 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 94 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 98 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 100 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 101 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 102 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 103 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 104 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 106 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 108 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 110 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 111 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 115 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 117 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 118 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 122 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 124 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 126 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 130 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 132 was not found when attempting to remove it
  GLib.source_remove (self.timer)
/usr/share/system-config-printer/applet.py:358: Warning: Source ID 138 was not found when attempting to remove it
  GLib.source_remove (self.timer)
